azplugins/BondEvaluatorFENE.h

```diff
@@ -98,10 +127,20 @@ class BondEvaluatorFENE
                 force_divr = r2inv * r6inv * (Scalar(12.0)*lj1*r6inv - Scalar(6.0)*lj2);
                 bond_eng = r6inv * (lj1*r6inv - lj2) + epsilon;
                 }
+            // Check if delta is non -zero,if non-zero sqrt of rsq is calculated
+            if (delta!=Scalar(0))
```

Comment:
Put whitespace around binary operators:
```suggestion
            if (delta != Scalar(0))
```